lib/Language/Haskell/Stylish/Step/LanguagePragmas.hs

```diff
@@ -87,10 +83,14 @@ truncateComma xs
 
 
 --------------------------------------------------------------------------------
-prettyPragmas :: Int -> Int -> Bool -> Style -> [String] -> Lines
-prettyPragmas _    longest align Vertical    = verticalPragmas longest align
-prettyPragmas cols _       _     Compact     = compactPragmas cols
-prettyPragmas cols _       align CompactLine = compactLinePragmas cols align
+prettyPragmas :: String -> Int -> Int -> Bool -> Style -> [String] -> Lines
+prettyPragmas lp _    longest align Vertical    = verticalPragmas lp longest align
+prettyPragmas lp cols _       _     Compact     = compactPragmas lp cols
+prettyPragmas lp cols _       align CompactLine = compactLinePragmas lp cols align
+
+--------------------------------------------------------------------------------
+makeLang :: String -> String
+makeLang x = if fmap toLower x == "language" then x else "LANGUAGE"
```

Comment:
And then this shouldn't be necessary anymore, which I think is nice because erroring out is usually better than implicitly changing what the user wanted